control-plane/tests/test_service_persistence.py

```
@@ -7750,6 +7803,935 @@ def test_service_executes_phase20_first_live_action_end_to_end_from_reviewed_rec
+
+        with self.assertRaisesRegex(
+            ValueError,
+            "action execution 'action-execution-phase21-expiry-window-001' exceeds approval expiry binding",
+        ):
+            restored_service.restore_authoritative_record_chain_backup(backup)
+        self._assert_authoritative_store_empty(restored_store)
+
+    def test_service_phase21_restore_rejects_reconciliation_run_binding_mismatch(
+        self,
+    ) -> None:
+        _store, service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.9)

[warning] Unpacked variable `reviewed_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)